braces/views/_access.py

```diff
@@ -392,3 +396,20 @@ def dispatch(self, request, *args, **kwargs):
 
         return super(StaffuserRequiredMixin, self).dispatch(
             request, *args, **kwargs)
+
+
+class RecentLoginRequiredMixin(LoginRequiredMixin):
+    """
+    Mixin allows you to require a login to be within a number of seconds.
+    """
+    max_last_login_delta = 1800  # Defaults to 30 minutes
```

Comment:
Might think about adding a get method for this so that it can be overridden. Someone might want to perform some logic and set the amount of time based on that.